purchase.adoc

```diff
@@ -388,288 +517,533 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
+Items row discounts will be applied first, followed by the discount on the sum of the already discounted row prices.
 
-When applying the total discounts on the purchase price, the row discounts will first be applied, followed by the cart discount which is then applied on the sum of the already discounted row prices.
+*VAT calculation on purchase with discounts*
 
-#### VAT calculation when discounts have been applied
-VAT is calculated after all the discounts have been applied. Since each row can have a separate VAT percentage the cart discount is evenly distributed over all row of items before the VAT per row of products can be calculated.
+Since each row of items can have a separate VAT percentage the discount
+for the whole purchase is evenly distributed over all purchased items.
+VAT is calculated after all the discounts are deducted.
 
-If the cart discount is a fixed value then that value will be deducted from each row according the price of each row in relation to the total purchase price.
+If the purchase discount is a fixed amount then that amount  will be deducted from each row according the price of each row in relation to the total purchase price.
 
-##### Examples
-If the cart discount is 20%, then the price of each row of item will be deducted with 20% before calculating the VAT for each row.
+*Examples*
 
-If a purchase has a total price of 100 and contains two rows with the first row priced at 40 and the second at 60, then 40% of the fixed cart discount will be deducted from the first row and 60% from the other row before calculating the VAT.
+If the purchase discount is 20%, then the price of each row of item will be deducted with 20% before calculating the VAT for each row.
 
-#### Response structure
-Row discounts are expressed using the following JSON structure in the `products` list, containing either `amount` for fixed discounts or `percentage` for percentage discounts. The `discountValue` field following the `discount` structure contains the total discount amount for the row:
-```json
+If a purchase has a total price of 100 and contains two rows with the first row priced at 40 and the second at 60, then 40% of the fixed purchase discount will be deducted from the first row and 60% from the other row before calculating the VAT.
+
+* Row discounts are expressed using the following JSON structure in the `products` list, containing either `amount` for fixed discounts or `percentage` for percentage discounts.
+The `discountValue` field following the `discount` structure contains the total discount amount for the row:
++
+--
+[source,json]
+----
 "discount": {
-  "quantity": 1,
-  "percentage": 10
+    "name": "10% off",
+    "quantity": 1,
+    "percentage": 10
 },
 "discountValue": 1000
-```
+----
+
 *Note:* Currently, we only support one discount per row of items, so the `quantity` field will always be set to 1. This could potentially change in the future.
```

Comment:
Usually the first person pronouns like we and I are not used in instructional tech docs, except in blog posts and marketing materials.